src/test/ui/macros/macro-at-most-once-rep-2018-feature-gate.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Feature gate test for macro_at_most_once_rep under 2018 edition.
+
+// gate-test-macro_at_most_once_rep
+// compile-flags: --edition=2018
```

Comment:
unrelated, but we should really add a `// edition: 2018` option, shouldn't we?

Reply:
Created #51800

Reply:
Done!